docs/visualization/plotting-overview.ipynb

```diff
@@ -0,0 +1,256 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Plotting Overview\n",
+    "\n",
+    "## Getting started\n",
+    "\n",
+    "`scipp` offers a number of different ways to plot data from a `DataArray` or a `Dataset`.\n",
+    "It uses the `matplotlib` graphing library to do so, as well as the `ipyvolume` project for 3D visualizations."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import scipp as sc\n",
+    "from scipp import Dim"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "N = 20\n",
+    "M = 5\n",
+    "d = sc.Dataset()\n",
+    "d.coords[Dim.X] = sc.Variable([Dim.X], values=np.arange(M), unit=sc.units.m)\n",
+    "d.coords[Dim.Y] = sc.Variable([Dim.Y], values=np.arange(N), unit=sc.units.us)\n",
+    "\n",
+    "d['data'] = sc.Variable([Dim.Y],\n",
+    "                        values=100*np.random.rand(N)+50,\n",
+    "                        unit=sc.units.counts)\n",
+    "d['data_with_errors'] = sc.Variable([Dim.Y],\n",
+    "                                    values=50*np.random.rand(N),\n",
+    "                                    variances=50*np.random.rand(N),\n",
+    "                                    unit=sc.units.counts)\n",
+    "\n",
+    "d['data_2d'] = sc.Variable([Dim.X, Dim.Y],\n",
+    "                           values=10.0*np.random.rand(M, N))"
```

Comment:
Maybe adding a unit to the 2d data would be good, as it would also show up next to the colorbar?